observability-test/spanner.ts

```diff
@@ -860,9 +867,11 @@ describe('ObservabilityOptions injection and propagation', async () => {
         traceExporter.reset();
 
         tx!.begin();
-        tx!.runUpdate(updateSql, (err, rowCount) => {
+        tx!.runUpdate(updateSql, async (err, rowCount) => {
           assert.ifError(err);
+          tx!.end();
 
+          await tracerProvider.forceFlush();
           traceExporter.forceFlush();
```

Comment:
traceExporter.forceFlush is also a promise, why not await on it?

Reply:
Could add one but it's less of a synchronization problem for traceExporter as it uses a simple span processor interface that immediately bundles in spans as they are ready to go from the tracerProvider.